lib/opensusebasetest.pm

```diff
@@ -271,14 +275,14 @@ Also see poo#18016 for details.
 =cut
 sub rewrite_static_svirt_network_configuration {
     my ($self) = @_;
-    type_line_svirt "root", expect => 'Password';
-    type_line_svirt "$testapi::password";
+    type_line_svirt("root", expect => 'Password');
+    type_line_svirt("$testapi::password");
     my $virsh_guest = get_required_var('VIRSH_GUEST');
-    type_line_svirt "sed -i \"\\\"s:IPADDR='[0-9.]*/\\([0-9]*\\)':IPADDR='$virsh_guest/\\1':\\\" /etc/sysconfig/network/ifcfg-\*\"", expect => '#';
+    type_line_svirt("sed -i \"\\\"s:IPADDR='[0-9.]*/\\([0-9]*\\)':IPADDR='$virsh_guest/\\1':\\\" /etc/sysconfig/network/ifcfg-\*\"", expect => '#');
     type_string "# output of current network configuration for debugging\n";
-    type_line_svirt "cat /etc/sysconfig/network/ifcfg-\*", expect => '#';
-    type_line_svirt "systemctl restart network",           expect => '#';
-    type_line_svirt "systemctl is-active network",         expect => 'active';
+    type_line_svirt("cat /etc/sysconfig/network/ifcfg-\*", expect => '#');
+    type_line_svirt("systemctl restart network",           expect => '#');
+    type_line_svirt("systemctl is-active network",         expect => 'active');
```

Comment:
I was changing that because it's a best practice and IMHO, the file shouldn't have the mixed coding styles all over it. ( 99% of the file is invoking subroutines in parenthesis)

Anyway, i reverted those 4 changed lines. I'll create another commit, with those changes.